operator/src/main/resources/scripts/stopServer.sh

```diff
@@ -11,7 +11,7 @@
 
 SCRIPTPATH="$( cd "$(dirname "$0")" > /dev/null 2>&1 ; pwd -P )"
 source ${SCRIPTPATH}/utils.sh
-[ $? -ne 0 ] && echo "Error: missing file ${SCRIPTPATH}/utils.sh" && exit 1
+[ $? -ne 0 ] && echo "[SEVERE] Missing file ${SCRIPTPATH}/utils.sh" && exit 1
```

Comment:
I assume we can't use trace here for some reason?

Reply:
Correct.  This echo is reporting that the we couldn't load the utils.sh file which contains the 'trace' function.